game-core/src/main/java/games/strategy/engine/data/UnitsList.java

```diff
@@ -14,10 +14,10 @@
  */
 public class UnitsList implements Serializable, Iterable<Unit> {
   private static final long serialVersionUID = -3134052492257867416L;
-  // maps GUID -> Unit
   // TODO - fix this, all units are never gcd
```

Comment:
Oh and this might be a good thing to fix, would reduce memory usage by a lot on longer games.